docs/connector-development/README.md

```diff
@@ -165,32 +161,22 @@ Once you've finished iterating on the changes to a connector as specified in its
    
    * Then run the command `./gradlew :airbyte-config:init:processResources` to generate the seed spec yaml files, and commit the changes to the PR. See [this readme](https://github.com/airbytehq/airbyte/tree/master/airbyte-config-oss/specs-oss) for more information.
 
-5. The new version of the connector is now available for everyone who uses it. Thank you!
+5. Once the PR is merged the new connector version will be published to DockerHub and the connector should now be available for everyone who uses it. Thank you!
 
 
 ### Updating Connector Metadata
 
-When a new (or updated version) of a connector is ready to be published, our automations will check your branch for a few things:
+When a new (or updated version) of a connector is ready, our automations will check your branch for a few things:
 * Does the connector have an icon?
 * Does the connector have documentation and is it in the proper format?
 * Does the connector have a changelog entry for this version?
 
-If any of the above are failing, you won't be able to merge your PR or publish your connector.
+If any of the above are failing, you won't be able to merge your PR.
 
 Connector icons should be square SVGs and be located in [this directory](https://github.com/airbytehq/airbyte/tree/master/airbyte-config-oss/init-oss/src/main/resources/icons).
 
 Connector documentation and changelogs are markdown files which live either [here for sources](https://github.com/airbytehq/airbyte/tree/master/docs/integrations/sources), or [here for destinations](https://github.com/airbytehq/airbyte/tree/master/docs/integrations/destinations).
 
-### The /publish command
```

Comment:
since normalization people are still using this, could we keep this reference around somewhere? maybe just in the normalization README?